app/pages/lab/project-details.cjsx

```diff
@@ -30,6 +30,24 @@ module.exports = React.createClass
     backgroundError: null
```

Comment:
This and `avatarError` can be removed from the state in favor of the single handler.